### ISSUE TITLE
National Blackout Error

### ISSUE THREAD

**Issue by SlimJimPoisson** [kmac/mlbv#15]
I live outside the US which means that I can receive live games that are blacked out nationally, but when trying to use mlbv I get "Could not load stream {'errors': ['blackout']}".

**Comment by kmac** [comment]
Odd. I'm not in the US either. I will have to try this.

**Comment by SlimJimPoisson** [comment]
I wanted to be clear about my experience. As you might guess I was attempting to access the Sox-Yankees game from Sunday night that aired on ESPN. I could access the game through MLB.tv using my browser, but using mlbv caused the error reported above.

My command line was
mlbv --yesterday --team bos --inning 1 --fetch --resolution 720p --feed nat

I use "--yesterday" because I am living in Asia where yesterday is today.

**Comment by SlimJimPoisson** [comment]
My bad, I know what I did wrong. You can close this one as "user error".

**Comment by kmac** [comment]
Okay, thanks for letting me know.

**Comment by SlimJimPoisson** [comment]
OK, it appears that we need to reopen this one. I suggested that this issue be closed based on discovering faulty testing parameters, but I did not retest after that assuming that the tool would work. I have now discovered that even with what I think is a correct usage I am still receiving this error. Here is the command line (though I have tried several variants) and the complete output:

`mlbv --yesterday --fetch --resolution 720p --team nyy --feed nat`

Results:
Could not load stream
{'errors': ['blackout']}
No stream URL found

In reviewing I found an "already logged in" message from session.py. It made me wonder if something about my location is being retained in the session. I have no idea if I have found the problem, but is there a way to logout (either through python or externally)?
